heuristic.py

```diff
@@ -1,21 +1,66 @@
-from converter import convert
-from destroy_algorithms import *
-from heuristic_calculations import *
-from model.feasibility_model import FeasibilityModel
-from model.optimality_model import OptimalityModel
-from repair_algorithms import *
+from model.model_class import Optimization_model
+from model.hard_constraint_model_class import Optimization_model as Feasibility_model
+from heuristic.heuristic_calculations import calculate_objective_function as calc_ob
+from heuristic.criterions.greedy_criterion import GreedyCriterion
+from heuristic.criterions.simulated_annealing_criterion import SimulatedAnnealingCriterion
+from heuristic.state import State 
+from heuristic.alns_calculator import Alns_calculator
+from heuristic.converter import convert
+from heuristic.alns import ALNS
+from heuristic.heuristic_calculations import *
+import cProfile
+import pstats
 
 
 def main():
```

Comment:
Is the intention of this file to have a easier way to profile the ALNS?

Reply:
This is the file I am using when running. It because I feel it is easier to use.